### PR TITLE
chore(e2e): add readme

### PR DESCRIPTION
**Pull request by kuzhelov**
This PR adds `README.md` file which summarises main points related to end-to-end tests' workflow and organization.

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Stardust E2E tests
+This directory provides end-to-end (in-browser) tests for critical scenarios for Stardust.
+All test scripts, as well as examples this scripts are running against, are contained in `./tests` directory.
+
+## Run browser tests
+The following `yarn` scripts provided to deal with browser tests:
```
Comment (mnajdova) [inline e2e/README.md]:
```suggestion
The following `yarn` scripts are provided to deal with browser tests:
```

File: e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- `yarn test:e2e` - independent task: builds, serves and runs tests. Supports flags:
+    - `--skip-build` - disables prior (re)build of Docs site which end-to-end tests are running against.
+- `yarn test:e2e:serve` - builds and serves site with E2E examples. Index page of this site contains link to all E2E examples.
+
+## Add browser test
+Lets suppose that one would like to introduce tests for some scenario with `<scenarioName>`. This is set of steps to follow:
```
Comment (mnajdova) [inline e2e/README.md]:
```suggestion
Lets suppose that one would like to introduce tests for some scenario with `<scenarioName>`. This is the set of steps to follow:
```

File: e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+     - this `gotoTestCase` function will trim `-(example|test).tsx?` postfix from file name, to leave only `<scenarioName>` part
+
+## Use browser API
+There is global `e2e` object provided to each test function that serves as a thin wrapper on top of `puppeteer's` API. This `e2e` object should be used as a single client's entry point to simulate browser stuff: clicks, focus and keyboard events.
+
+## How browser tests work
+This is what happens when `yarn test:e2e` script is executed:
+- test files in `./tests` directory are discovered
+- bundled, all the bundle artifacts go to `./dist` directory
+- server starts on dedicated for end-to-end tests port, it serves from `./dist`
```
Comment (mnajdova) [inline e2e/README.md]: Let's add the actual port here.

Comment (kuzhelov) [reply e2e/README.md]: have updated with the information that helps to discover this port's value - at the same time, won't like to duplicate the one provided in `config` file, to keep things easier to maintain

**Review by mnajdova — APPROVED** [review]
Looks good, thanks for adding this.

**Pull request merged**